assets/src/edit-story/app/rightClickMenu/test/useRightClickMenu.js

```diff
@@ -161,7 +161,36 @@ describe('useRightClickMenu', () => {
   });
 
   describe('Text element right clicked', () => {
-    it.todo('should return the correct menu items');
+    beforeEach(() => {
+      mockUseStory.mockReturnValue({
+        ...defaultStoryContext,
+        selectedElements: [
+          {
+            id: '1',
+            type: 'text',
+          },
+        ],
+      });
+    });
+
+    it('should return the correct menu items', () => {
+      const { result } = renderHook(() => useRightClickMenu(), {
+        wrapper: RightClickMenuProvider,
+      });
+
+      const labels = result.current.menuItems.map((item) => item.label);
+      expect(labels).toStrictEqual([
+        ...expectedDefaultActions,
+        RIGHT_CLICK_MENU_LABELS.SEND_BACKWARD,
+        RIGHT_CLICK_MENU_LABELS.SEND_TO_BACK,
+        RIGHT_CLICK_MENU_LABELS.BRING_FORWARD,
+        RIGHT_CLICK_MENU_LABELS.BRING_TO_FRONT,
+        RIGHT_CLICK_MENU_LABELS.COPY_STYLES,
+        RIGHT_CLICK_MENU_LABELS.PASTE_STYLES,
+        RIGHT_CLICK_MENU_LABELS.ADD_TO_TEXT_PRESETS,
+        RIGHT_CLICK_MENU_LABELS.ADD_TO_COLOR_PRESETS,
```

Comment:
soooo many actions 😲

Reply:
yea I the background image actions feel a little bloated as well

Reply:
These menus are massive. On small screens there's no way that users can see all of the items unless the click in the right place.

Max brought this up last week too - we could see later if we want to scroll the menus or reduce the number of actions. Regardless, something to chat with omar and will about later.

Reply:
totally agree! it's good to see it all together.